Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:jammy-20220421
+
+#ENV REFRESHED_AT 2022-04-28 # clear cache
+
+RUN apt-get -qq update \
+        && apt-get -y install scons python3.10 wget python3-pip
+
+COPY requirements.txt /firmware/
```

Comment:
don't put this in `/firmware` or else it will conflict when you mount the volume later. Just put it in `/requirements.txt`